third_party/txt/src/txt/font_collection.cc

```diff
@@ -30,6 +30,17 @@
 
 namespace txt {
 
+namespace {
+
+// Example characters representing character classes that may require a
+// fallback font.
+const std::vector<SkUnichar> fallback_characters{
+    0x1f600,  // emoji
+    0x4e00,   // CJK
```

Comment:
the CJK case here won't be a long-term solution. What font to use when rendering CJK characters depends on the locale.